index.html

```diff
@@ -102,7 +102,7 @@
 <body>
 <h1 class="title p-name" id="title">Requirements for Chinese Text Layout<br/><span lang="zh">中文排版需求</span></h1>
 <div id="abstract">
-  <p its-locale-filter-list="en" lang="en"> This document summarizes the text composition requirements in the Chinese writing system. One of the goals of the task force is to describe the issues in the Chinese layout requirements, another one is to provide satisfactory equivalent to the current standards (i.e. Unicode), also to promote vendors to implement those relevant features correctly.</p>
+  <p its-locale-filter-list="en" lang="en"> This document summarizes the text composition requirements in the Chinese writing system. One of the goals of the task force is to describe the issues in the Chinese layout requirements, another one is to provide satisfactory equivalent to the current standards (e.g., Unicode), also to promote vendors to implement those relevant features correctly.</p>
```

Comment:
Suggested edits indicated by ***:

This document summarizes *** text composition requirements in the Chinese writing system. One of the goals of the task force is to describe *** issues ***for Chinese layout***, another *** is to provide ***a satisfactory equivalent to *** current standards (***such as Unicode), ***as well as to ***encourage vendors to implement *** relevant features correctly.


I'm not entirely sure what "provide a satisfactory equivalent to" means.

Reply:
Thanks @r12a. I just pushed a commit using your text, and changed "provide satisfactory equivalents to the current standards" to "describe correspondence with existing standards" (since this seems to be what the Chinese text means).

Reply:
@xfq, looking at the files changed tab, it seems like some of the changes made in the copy of the text lower down (which i suspect is in the intro (?) were not made in this location.

Reply:
Ah, indeed. I have sync'd the text in those two places now.